examples/server/tests/tests.sh

```diff
@@ -6,9 +6,18 @@ cd $SCRIPT_DIR
 
 set -eu
 
+if [[ "${SLOW_TESTS:-0}" == 1 ]]; then
```

Comment:
```suggestion
if [[ "${SLOW_TESTS:-0}" == 1 ]]; then
```

Small nits, personally I always messy up with `VARIABLE=1`, `VARIABLE=ON` and `VARIABLE=YES` 😂  , so probably just check if the variable exists here without verifying its value